docker_environment/Dockerfile

```
@@ -59,7 +60,11 @@ RUN ln -sfn /usr/bin/python3.6 /usr/bin/python & \
     ln -s /usr/bin/pip3 /usr/bin/pip
 
 # Upgrade pip
-RUN pip install --upgrade pip
+RUN pip3 install --upgrade pip
```

I'd suggest we use fixed matching versions for both `pip`, `setuptools` and `wheels`, such as the following ones :

```suggestion
RUN pip3 install pip==21.0.1 setuptools==53.0.0 wheel==0.36.2 --no-cache-dir
```